dist/index.html

```diff
@@ -3,39 +3,32 @@
 <head>
 	<meta charset="UTF-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<title>Online Smoke Shop</title>
+	<title>Pokemon Desk</title>
 	<link rel="stylesheet" href="style.css">
     <script src="https://kit.fontawesome.com/286c39d0b9.js" crossorigin="anonymous"></script>
 <script defer src="main.js"></script></head>
 <body>
 <!-- Header Section -->
 <header id="header">  
-<div class="navbar">
-    <img src="img\pokeball.png" alt="Logo" class="logo">
-    <button class="hamburger">
-        <span class="bar"></span>
-        <span class="bar"></span>
-        <span class="bar"></span>
-    </button>
-</div>
-<!-- Overlay Section -->         
-<div class="overlay">
-    <button class="close-button">&times;</button>
-    <ul id="overlay-nav">
-        <li><a href="#" class="navigation">Characters</a></li>
-        <li><a href="#" class="navigation">Elements</a></li>
-        <li><a href="#" class="navigation">Abilities</a></li>
-        <li><a href="#" class="navigation">Levels</a></li>
-    </ul>
-</div>
+	<div class="navbar">
+		<img src="img\pokeball.png" alt="Logo" class="logo">
+		<div class="menu">
+			<ul id="main-nav">
+				<li><a href="#" class="navigation">Characters</a></li>
+				<li><a href="#" class="navigation">Elements</a></li>
+				<li><a href="#" class="navigation">Abilities</a></li>
+				<li><a href="#" class="navigation">Levels</a></li>
+			</ul>
+		</div>
+	</div>
 </header>
 <!-- Pokemon Cards -->
 <div id="item-container"></div>
 <!-- Footer Section --> 
 <footer>
-    <p>&copy; Created by Nurbol, Jose and Cosmos 2023</p>
+	<p>&copy; Created by Nurbol, Jose and Cosmos 2023</p>
 </footer>
-</section>
+<button id="scroll-to-top-btn"><i class="fas fa-arrow-up"></i></button>
 <script src="index.js"></script>
```

Comment:
You have done very impressive work, however since we are using webpack, we can ignore linking script files and css files in our index.html page.